src/layout/Bounds.jsx

```diff
@@ -1,16 +1,37 @@
-import PropTypes from 'prop-types';
+// @flow
+
 import React from 'react';
 import cx from 'classnames';
 
 import withLoading from '../utils/components/withLoading';
 
 export const BOUNDS_CLASS = 'bounds';
 
+type Props = {
+	/** The child elements of the component */
+	children: React$Node,
+	className?: string,
+	/** Whether the bounds max-width should use the narrow variant */
+	narrow?: boolean,
+	/** Props to pass to the `<Loading />` component */
+	loadingProps?: {
+		color?: string,
+		scrimColor?: string,
+		size?: MediaSizes,
+	},
+	/** Whether the component is in a loading state */
+	isLoading?: boolean,
+};
+
+type State = {};
 /**
  * Design System Component: Provides `bounds` container for components
  * @module BoundsComponent
  */
-export class BoundsComponent extends React.Component {
+
+export class BoundsComponent extends React.PureComponent<Props, State> {
+	state = {};
```

Comment:
How come `state` is being added to the component?

Reply:
OH i forgot to remove this, I was trying to clean up some warnings from storybook